src/pages/test-report/test-report.html

```diff
@@ -1,31 +1,372 @@
 <ion-header>
-  <ion-navbar>
-    <ion-title>Test Report</ion-title>
+  <ion-navbar hideBackButton>
+    <ion-title>Test Report - {{pageState.candidateUntitledName$ | async}}</ion-title>
     <ion-buttons end>
-      <button ion-button (click)="logout()">Logout</button>
+      <button ion-button navPush="DebriefPage">End test</button>
     </ion-buttons>
   </ion-navbar>
 </ion-header>
 
-<ion-content padding>
-  <ion-grid no-padding>
+<ion-content no-padding no-bounce>
+  <div style="background: white; height: 68px; padding: 0">
```

Comment:
Inline styles?

Reply:
It's a temporary box to give us the correct height for the toolbar , it will be removed when we add the toolbar